src/components/view-pane/index.js

```diff
@@ -118,20 +117,20 @@ class ViewPane extends Component {
   renderContent () {
     const {
       item,
+      onAddValue,
       onDeleteValue,
       onButtonClick,
       onChange,
       height,
       items
     } = this.props;
-    console.log('ONDELETEVALUE', onDeleteValue);
     if (item) {
       if (this.state.activeKey === 1) {
         return (
           <div>
             {this.renderTabs()}
             {this.renderTagContentHeader(item)}
-            {this.renderMetadataContent(item, onDeleteValue, height)}
+            {this.renderMetadataContent(item, onDeleteValue, onAddValue, height)}
```

Comment:
is it necessary to pass in these as arguments to the function? you could access the values from this.props directly inside the method.